geekhours/command.py

```diff
@@ -42,6 +42,30 @@ def remove_donelist(self, date: str, course: str):
         """ Call database.remove_donelist() """
         self.database.remove_donelist(date, course)
 
+    def show_total_hours(self):
+        """ Call database.get_total_hours() """
+        return self.database.get_total_hours()
+
+    def show_total_hours_course(self):
+        """ Call database.get_total_hours_course() """
+        return self.database.get_total_hours_course()
+
+    def show_total_hours_week(self, course: str = None):
```

Comment:
This method should replace a number for days of the week with a readable name.  A user would be hard to understand what number means.


0 = Sunday or Sun
1 = Monday or Mon
2 = Tuesday or Tue
...

Reply:
Added `name_months_and_days()` to replace  the numbers of the month and day of the week to readable names.
Used this method in `show_total_hours_month()` and  `show_total_hours_week()` to fix above issue.